### ISSUE TITLE
Intermittently diffing FSI threaded test

### ISSUE THREAD

**Issue by dschwen** [idaholab/moose#27555]
### Motivation

Threaded FSI testing diffs intermittently (e.g. here: https://civet.inl.gov/job/2212453/) on 

```
test:2d-finite-strain-steady.test: ERROR: In file thermal-me.csv: The values in column "max_solid_Ts" don't match. 
test:2d-finite-strain-steady.test: 	relative diff:   1.673e+03 ~ 1.673e+03 = 1.002e-04 (1.002e-04)
```

### Design

This might just be a tolerance issue, but somebody more familiar with FSI should take a closer look.

### Impact

Avoid having to invalidate a 2.5h test recipe.

**Comment by hugary1995** [comment]
Added two new timeouts to the end of the list.

**Comment by lindsayad** [comment]
I have a commit disabling that test in debug mode in https://github.com/idaholab/moose/pull/31268

**Comment by hugary1995** [comment]
A new one
```
[299.0s] [ 319MB]  TIMEOUT mfem/lor.MFEMCurlCurlLOR FAILED (TIMEOUT) [pedantic check]
```
https://civet.inl.gov/job/3443518/
@nmnobre

**Comment by hugary1995** [comment]
Another mfem LOR test is also pretty heavy (but not heavy enough to trigger a timeout):
```
[173.6s] [ 452MB]       OK mfem/lor.MFEMGradDivLOR [pedantic check]
```

And a few json outputting tests are on the heavy end as well
```
[171.8s] [ 732MB]       OK outputs/format.json_no_template [pedantic check,FINISHED]
[180.9s] [ 721MB]       OK outputs/format.hit [pedantic check,FINISHED]
[172.7s] [ 640MB]       OK outputs/format.json_line_info [pedantic check,FINISHED]
[157.8s] [ 550MB]       OK outputs/format.hit_search [pedantic check,FINISHED]
[159.3s] [ 532MB]       OK outputs/format.json_search [pedantic check]
[172.1s] [ 389MB]       OK outputs/format.json_full [pedantic check]
[167.2s] [ 372MB]       OK outputs/format.json_no_test_objects [pedantic check]
```
IDK if this is partly due to filesystem IO, but I think they are worth shortening to prevent future timeouts.

**Comment by nmnobre** [comment]
[This](https://github.com/idaholab/moose/pull/31906#issuecomment-3614901968) comment aged like milk I see xD

**Comment by hugary1995** [comment]
Ah, okay. I didn't know you've already addressed it.

**Comment by nmnobre** [comment]
> Ah, okay. I didn't know you've already addressed it.

No, on the contrary, I convinced @lindsayad to leave those tests alone, but I clearly shouldn't have. 😅

**Comment by hugary1995** [comment]
LOL I misread the conclusion there. Yeah, the CIVET machines sometimes run ~5x slower than my workstation, in my experience. It's also worse for DEBUG runs, apparently.

**Comment by lindsayad** [comment]
I would say you convinced me to leave them alone for the moment, which I was fine with since getting that package update PR over the finish line was already a huge PITA. Would be great to see those removed from debug testing or shortened

**Comment by GiudGiud** [comment]
Added two more tests to the list, one fixed now.
